osh/hub/scan/messaging.py

```diff
@@ -71,8 +72,9 @@ def send_message(message, key):
     this function sends specified message to broker and append specified
         key to ROUTING_KEY
     """
-    s = SenderThread(key, message)
-    s.start()
+    for topic in settings.UMB_TOPIC_PREFIX_LIST:
```

Comment:
This needs to be implemented in a backward-compatible way so that updated code works with existing configuration.  I would keep the name `UMB_TOPIC_PREFIX` and just extend its semantics such that it can be either a string or a list of strings.  Apart from backward-compatibility, an advantage is that you do not need to document what will happen when both `UMB_TOPIC_PREFIX` and `UMB_TOPIC_PREFIX_LIST` are set.

Reply:
I modified the semantics so it is backwad-compatible with existing configuration.

Let me know please if that is what you meant. 

Thank you!

Reply:
@kdudka I know that you want your upgrades as seamless as possible but I'd like to see this hack removed after some time in the future.  The code of OSH is already excessively convoluted as it is.